apps/src/templates/sectionProgress/sectionProgressLoader.js

```diff
@@ -22,7 +22,7 @@ import logToCloud from '@cdo/apps/logToCloud';
 
 const NUM_STUDENTS_PER_PAGE = 20;
 
-export function loadScriptProgress(scriptId, sectionId) {
+export function loadUnitProgress(scriptId, sectionId) {
```

Comment:
Script => unit

This was a relatively recent name change.

Reply:
Thanks for changing this and leaving it better than how you found it!